package.json

```
@@ -39,7 +39,8 @@
     "dev": "NODE_ENV=development BROWSERSYNC_OPEN=true gulp dev",
     "docs:mod": "node tasks/mod-extractor.js",
     "lint:components": "node ./tasks/packageLint.js",
-    "mv:preview": "rimraf dist/preview && mv -f tools/preview/storybook-static dist/preview",
+    "lint:styles": "stylelint ./components/**/*.css",
```

```suggestion
    "lint:styles": "stylelint ./components/*/dist/**/*.css",
```

^ Try this command - it's still running on pre-compiled assets b/c it's picking them up from the `components/*/*.css`. We can target dist specifically or we can look into ways to ignore the postcss specific syntax that's tripping up the linter.